index.md

```diff
@@ -261,9 +261,10 @@ _If there's one thing the world needs, it's more todo apps!_ 👍
 
 ## Project setup
 
-### 1. Clone the Wisemen NestJS Template from Github
+### 1. Fork the Wisemen NestJS Template from Github
```

Comment:
Waarom fork?

Reply:
Omdat we wouden voorkomen dat elke persoon die de onboarding doet een branch aanmaakt op het template project. Dus forken naar je eigen repo leek een goede optie. What do you think?

Reply:
Slim, helemaal top!